ibc/utils_test.go

```diff
@@ -219,22 +219,32 @@ func TestGetReceivedCoin(t *testing.T) {
 			"channel-2",
 			"transfer/channel-0/transfer/channel-1/uatom",
 			"10",
-			sdk.Coin{Denom: precompilestestutil.UatomIbcdenom, Amount: math.NewInt(10)},
+			sdk.Coin{Denom: precompilestestutil.UatomIbcDenom, Amount: math.NewInt(10)},
 		},
 		{
 			"transfer ibc wrapped coin to destination which is not its source",
 			"transfer",
-			"channel-0",
+			"channel-2",
```

Comment:
- What was the reason for making this change?
- The test case above has been switched to a different scenario than described, but the description hasn’t been updated, which could cause confusion.